tailwind.config.js

```diff
@@ -47,6 +49,13 @@ module.exports = {
           'monospace',
         ],
       },
+      padding: {
+        yt: '56.25%',
```

Comment:
do not add specific padding values in percentage. Use existing padding rem values

Reply:
I added It to make the youtube embed responsive ,using rem values won't solve the problem.
You can see in https://www.wallora.com/ that It will be overlap with other elements when screen size gets smaller.